### DISCUSSION TITLE
DietPi-Software | Ledger Live

### DISCUSSION THREAD

**Discussion by nodegraphics** [MichaIng/DietPi#5331] [Software Request]
As we know that there is no ARM support for Ledger Live.
![Screenshot - 2022-03-03 13 20 41](https://user-images.githubusercontent.com/10071439/156654792-cecb775b-636a-4445-a37c-055963d78cbe.png)

But I did noticed that there  is some attempts to make it work in ARM based Systems. 
[Ledobe](https://github.com/antichris/ledobe) > Ledger Live Dockerized Build Environment

I was wondering if it would be possible to include this in Dietpi Software list ? 

The reason i wanted to open this request is; 
Its very difficult for someone without expertise in linux and of what dockerized build environment or other details to install this software or even get close to make it work. ( Im one of that persons 🙋🏽)

**Comment by MichaIng** [comment]
Many thanks for your request.

Could you explain a bit more what it is about? If I understand it right, Ledger is a crypto wallet and Ledobe is a build environment which allows to build Ledger as Docker container, either directly on the host system or an image? There are two simple alternative commands given on the GitHub readme, are you running in issues using them? Of course Docker needs to be installed first, e.g. via `dietpi-software install 162`.

**Reply by nodegraphics** [reply]
> Many thanks for your request.
> 
> Could you explain a bit more what it is about? If I understand it right, Ledger is a crypto wallet and Ledobe is a build environment which allows to build Ledger as Docker container, either directly on the host system or an image? There are two simple alternative commands given on the GitHub readme,

Yes correct, its a crypto wallet which is widely used with hardware wallets nano x and nano s. I did not have much experience with Docker container and i couldn't run it myself after couple of tries. (specially tried in Debian + manjaro) 
But i noticed that Dietpi might be a perfect environment considering its usage and how lightweight it is. and considering the usage with arm based systems / boards.
This request might have a simple solution by installing docker environment in dietpi (which is available in dietpi-software list) and running ledger, but due to my lack of knowledge in Linux environment and not using docker container before made it impossible a bit...... i thought it might be good option to request a wallet that is used widely and still not available in any of repositories or  in software list. It might be a good option to have it in software list.


> are you running in issues using them? Of course Docker needs to be installed first, e.g. via `dietpi-software install 162`

Yes i did have issues using docker container to run ledger wallet in general. It was too technical considering my knowledge. I always felt that i wish some other distros had similar software listing just like dietpi, which makes everything easier on my end and let me focus on my main task as a designer.

I hope this request makes sense and might have potansial with its usage in community.

Thanks,
